api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -174,6 +174,19 @@ message TransformationTemplate {
   // resulting header, the rendered template will determine the value.
```

Comment:
Can we specify that existing headers will be replaced. Can we also explain the behavior if both `headers` and `headers_to_append` is specified.